photon-api/src/main/scala/com/linkedin/photon/ml/estimators/GameEstimator.scala

```diff
@@ -642,12 +652,13 @@ class GameEstimator(val sc: SparkContext, implicit val logger: Logger) extends P
     }
     val downSamplerFactory = DownSamplerHelper.buildFactory(task)
     val lockedCoordinates = get(partialRetrainLockedCoordinates).getOrElse(Set())
+    val indices = get(interceptIndices).getOrElse(Seq.empty[Int])
```

Comment:
Please remove this line - such code is best avoided by setting a default value and replacing the call with `getOrDefault`.